packages/core/core/src/services/document-service/transform/relations/transform/data-ids.ts

```diff
@@ -16,11 +16,14 @@ interface Options {
   allowMissingId?: boolean; // Whether to ignore missing ids and not throw any error
 }
 
+export const isPolymorphicRelation = (attribute: any): any =>
```

Comment:
Don't we already have something similar in `@strapi/utils+content-types`?

Reply:
(If not, maybe we could move this to the content types utils for the next person who need it?)

Reply:
I think this was copy-pasted from there to get it working, can probably just delete it and use the shared util

Reply:
absolutely! if we have a shared function for this we should reuse it, this was copy pasted from the database plugin, which does not export it